docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -765,6 +770,9 @@ The generated secret includes the following files:
 %prod.quarkus.http.tls-configuration-name=http
 %prod.quarkus.http.insecure-requests=disabled
 ----
+endif::no-kubernetes-secrets-or-cert-manager[]
+// The reason for this ifndef condition is that this content is not supported in product docs.
```

Comment:
This shouldn't be required/relevant here. The attributes could be listed somewhere describing what they capture.